sdks/python/setup.py

```diff
@@ -368,7 +368,7 @@ def get_portability_package_data():
           'grpcio>=1.33.1,!=1.48.0,<2',
           'hdfs>=2.1.0,<3.0.0',
           'httplib2>=0.8,<0.23.0',
-          'js2py>=0.74,<1',
+          'js2py>=0.74,<1; python_version<"3.12"',
```

Comment:
```suggestion
          # https://github.com/PiotrDabkowski/Js2Py/issues/317
          'js2py>=0.74,<1; python_version<"3.12"',
```

Reply:
Oh sorry I totally misunderstood what you were asking, thanks for adding that!